darts/tests/models/forecasting/test_global_forecasting_models.py

```diff
@@ -815,3 +821,122 @@ def test_residuals(self):
 
         res = model.residuals(ts)
         assert len(res) == 38 - (24 + 12)
+
+    @pytest.mark.parametrize(
+        "params",
+        product(
+            [
+                (
+                    {"series": sine_3_ts + 3},
+                    {"target": Scaler()},
+                ),
+                (
+                    {"series": sine_3_ts + 3, "past_covariates": sine_1_ts + 3},
+                    {"past": Scaler(scaler=MinMaxScaler())},
+                ),
+                (
+                    {"series": sine_3_ts + 3, "future_covariates": sine_1_ts + 3},
+                    {"future": Scaler(scaler=MaxAbsScaler())},
+                ),
+                (
+                    {
+                        "series": sine_3_ts + 3,
+                        "past_covariates": sine_2_ts + 3,
+                        "future_covariates": sine_1_ts + 3,
+                    },
+                    {"target": Scaler(), "past": Scaler()},
+                ),
+            ],
+            [True, False],
+        ),
+    )
+    def test_historical_forecasts_with_scaler(self, params):
```

Comment:
nice one 👍  :) 
Could you check for 2 historical forecast iterations? Also checking `last_points_only=False`, and `enable_optimization=True` would be important.